### ISSUE TITLE
Sound sources from the center

### ISSUE THREAD

**Issue by proyb6** [alemangui/pizzicato#41]
I wonder how did most recording could sound coming from the center to the left and right without harsh sounding? Limiter effect does the job and what other possible effects?
https://www.youtube.com/watch?v=A8LoaOjgn9E
https://www.youtube.com/watch?v=tuvWpVCUbr0

**Comment by alemangui** [comment]
I don't think I understand the question. Could you rephrase it? "center to the left and the right"... are you talking about stereo panning?

**Comment by proyb6** [comment]
Sorry, I mean to create a depth effect on a wave file you will hear the sound will come from the front center. It's similar to Audience mode (hear instruments from the stage) rather than Player mode.

**Comment by alemangui** [comment]
Audio spatialization is not included in Pizzicato, but you can definitely extend Pizzicato objects with other web audio nodes. 

That said, the AudioListener may be what you are looking for: https://developer.mozilla.org/en-US/docs/Web/API/AudioListener.
